charts/fleet-agent/values.yaml

```diff
@@ -25,11 +25,11 @@ clientID: ""
 clusterNamespace: ""
 
 # The namespace containing the clusters registration secrets
-systemRegistrationNamespace: fleet-clusters-system
+systemRegistrationNamespace: cattle-fleet-clusters-system
```

Comment:
This was necessary, because https://github.com/rancher/fleet/blob/master/pkg/namespace/util.go#L18-L24 generates this name. This is not a user facing namespace?

So if someone updates their fleet standalone installation they must pass `systemRegistrationNamespace` and `systemNamespace` and it should work.